test/koa-sslify-test.js

```diff
@@ -1,73 +1,77 @@
-var Koa = require('koa');
+var expect = require('chai').expect;
+var koa = require('koa');
 var agent = require('supertest-koa-agent');
 var enforce = require('../index.js');
 
-describe('HTTPS not enforced', () => {
+describe('HTTPS not enforced', function () {
 
-  var app = new Koa();
+  var app = koa();
```

Comment:
:D No problem. BTW this whole PR is targeted to koa2 branch. Please change it to `master`.